frontend/packages/core/src/AppLayout/utils.tsx

```diff
@@ -124,4 +126,27 @@ const searchIndexes = (workflows: Workflow[]): SearchIndex[] => {
   return indexOptions;
 };
 
-export { routesByGrouping, searchIndexes, sortedGroupings, workflowByRoute, workflowsByTrending };
+/** Filter out all of the workflows that are configured to be `hideNav: true`.
+ * This prevents the workflows from being discoverable by the user from the UI,
+ * both search and drawer navigation.
```

Comment:
may want to change this message now that it's on a function.
```suggestion
 * This prevents the workflows from being discoverable by the user from the UI where used.
 * Some example usages are in the search and drawer navigation components.
```

Reply:
Ah good call out, I completely forgot to update that comment